package.json

```diff
@@ -77,6 +77,19 @@
     "resolve": "^1.6.0"
   },
   "devDependencies": {
+    "@babel/helper-module-imports": "^7.0.0",
```

Comment:
This is going to cause a conflict for you...

Reply:
Huh, I guess not?!

Reply:
as long as the change is identical in both branches it's not a conflict 😉